stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -730,7 +730,7 @@ internal class ChatDomainImpl internal constructor(
 
     private suspend fun retryMessagesWithAttachments(): List<Message> {
         val retriedMessages = repos.selectMessagesWaitForAttachments()
-            .filter { message -> message.attachments.any { it.uploadState == Attachment.UploadState.InProgress } }
```

Comment:
I don't understand why removing solves this. You wrote in description that case is "attachments are already sent, but message isn't yet". But if attachment is sent it means that it has `uploadState == UploadState.Success`. And all attachments with `InProgress` state are already processing by worker